### DISCUSSION TITLE
Meridiano TV

### DISCUSSION THREAD

**Discussion by southerndream56** [iptv-org/community#998] [Q&A]
Can anybody help me with watching Meridiano TV from Venezuela ?
Thanks

**Comment by ztreamhd** [comment]
That's going to be difficult. That channel appears to be live streaming through Dailymotion (https://www.dailymotion.com/video/x8irn8c) so any links you extract from there will expire in around 24 hours. If you want a stable link to watch it, the only solution is to use or deploy a service like `query-streamlink` that grabs the M3U8 periodically.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19891
